app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,221 @@
+  height: 200px;
+
+  .btn {
+    margin-top: 130px;
+  }
+}
+
+#index-foods {
+  background: url("frog-in-pitcher-plant.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-plants {
+  background: url("venus_flytrap.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-accessories {
+  background: url("planters.jpg") no-repeat center center;
+  background-size: 100%;
+}
+
+#index-foods,
+#index-accessories,
+#index-plants {
+  height: 200px;
+}
+
+.img-portfolio {
+    margin-bottom: 30px;
+}
+
+.img-hover:hover {
+    opacity: 0.8;
+}
+
+/* Home Page Carousel */
+
+header.carousel {
+    height: 50%;
+}
+
+header.carousel .item,
+header.carousel .item.active,
+header.carousel .carousel-inner {
+    height: 100%;
+}
+
+header.carousel .fill {
+    width: 100%;
+    height: 100%;
+    background-position: center;
+    background-size: cover;
+}
+
+#ProductCarousel .carousel,
+#ProductCarouselSm .carousel {
+    margin-bottom: 0;
+    padding: 0 40px 30px 40px;
+}
+
+/* The controlsy */
+#ProductCarousel .carousel-control,
+#ProductCarouselSm .carousel-control {
+    height: 40px;
+  	width: 40px;
+    background: none repeat scroll 0 0 #222222;
+    border: 4px solid #FFFFFF;
+    border-radius: 23px 23px 23px 23px;
+    margin-top: 90px;
+}
+
+#ProductCarousel .carousel-control.right,
+#ProductCarouselSm .carousel-control.right {
+	right: -12px;
+	height: 40px;
+	width: 40px;
+  background: none repeat scroll 0 0 #222222;
+  border: 4px solid #FFFFFF;
+  border-radius: 23px 23px 23px 23px;
+  margin-top: 6%;
+	font-size: 20px;
+	color: #fff;
+}
+
+#ProductCarousel .carousel-control.left,
+#ProductCarouselSm .carousel-control.left {
+	left: -12px;
+	height: 40px;
+	width: 40px;
+  background: none repeat scroll 0 0 #222222;
+  border: 4px solid #FFFFFF;
+  border-radius: 23px 23px 23px 23px;
+  margin-top: 6%;
+	font-size: 20px;
+	color:#fff;
+}
+
+/* The indicators */
+#ProductCarousel .carousel-indicators,
+#ProductCarouselSm .carousel-indicators {
+	right: 50%;
+	top: auto;
+	bottom: 60px;
+	margin-right: -19px;
+}
+
+/* The colour of the indicators */
+#ProductCarousel .carousel-indicators li,
+#ProductCarouselSm .carousel-indicators li {
+	background: #cecece;
+}
+
+#ProductCarousel .carousel-indicators .active,
+#ProductCarouselSm .carousel-indicators .active {
+background: #428bca;
+}
+
+
+/* 404 Page Styles */
+
+.error-404 {
+    font-size: 100px;
+}
+
+/* Pricing Page Styles */
+
+.price {
+  display: block;
+  font-size: 50px;
+  line-height: 50px;
+}
+
+.price sup {
+  top: -20px;
+  left: 2px;
+  font-size: 20px;
+}
+
+.period {
+    display: block;
+    font-style: italic;
+}
+
+/* Footer Styles */
+
+footer {
+    margin: 50px 0;
+}
+
+/* Responsive Styles */
+
+@media(max-width:991px) {
+    .customer-img,
+    .img-related {
+        margin-bottom: 30px;
+    }
+}
+
+@media(max-width:767px) {
+    .img-portfolio {
+        margin-bottom: 15px;
+    }
+
+    header.carousel .carousel {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Avoid qualifying class selectors with an element.